kloppy/tests/test_statsbomb.py

```diff
@@ -68,6 +69,14 @@ def test_correct_deserialization(self):
             attacking_direction=AttackingDirection.NOT_SET,
         )
 
+        for qualifier in dataset.events[791].qualifiers:
+            if qualifier == BodyPartQualifier:
+                assert qualifier.value == "HEAD"
+
+        for qualifier in dataset.events[2231].qualifiers:
```

Comment:
This won’t fail when the qualifier is not set at all.

The get classifier by type might can help you here

Reply:
I must admit my test is a bit weak.

Do you mean the get_qualifier_value() function ? If so, I'll make the change

Reply:
Ah Yes that one. That way you can always do the assert. Which will fail in case the qualifier is not set or when it’s set incorrect

Reply:
Allright ! Thanks, I'll commit the changes